lrpar/src/lib/ctbuilder.rs

```diff
@@ -584,13 +584,17 @@ where
                         let mut cache = cache.to_string();
                         outc.retain(|c| !c.is_whitespace());
                         cache.retain(|c| !c.is_whitespace());
+                        #[cfg(grmtools_extra_checks)]
+                        const CACHE_HIT_EXPECTED: bool = std::env::var("CACHE_EXPECTED").is_ok();
+                        #[cfg(not(grmtools_extra_checks))]
+                        const CACHE_HIT_EXPECTED: bool = false;
                         if outc.contains(&cache) {
                             return Ok(CTParser {
                                 regenerated: false,
                                 rule_ids,
                                 conflicts: None,
                             });
-                        } else if std::env::var("CACHE_EXPECTED").is_ok() {
+                        } else if CACHE_HIT_EXPECTED {
```

Comment:
Might this be simpler as something like:

```rust
if outc.contains(...) { ... }
else {
   #[cfg(grmtools_extra_checks)]
   if std::env::vars("CACHE_EXPECTED").is_ok() { ... }
}
```

?

Reply:
Used that in [c0311e9](https://github.com/softdevteam/grmtools/pull/511/commits/c0311e9d4336bb89ffbce9cd7902fa4b09a80652)